### DISCUSSION TITLE
Possible to specify `babelfonts` on command line?

### DISCUSSION THREAD

**Discussion by max-heller** [jgm/pandoc#9313] [Q&A] [answered]
Is it possible to specify the [`babelfonts`](https://pandoc.org/MANUAL.html#fonts) variable from the command line? I can't figure out how to use `-V` to pass a map item

I've tried

- `-V babelfonts='chinese:"Noto Serif CJK"'`
- `-V babelfonts.chinese="Noto Serif CJK"`

**Comment by jgm** [comment, marked as answer]
No: the template expects `babelfonts` to be a key/value map, and only string values can be given on the command line. You could do it in the `variables` section of a defaults file, though.
